docs/podman-create.1.md

```diff
@@ -250,7 +250,17 @@ By default proxy environment variables are passed into the container if set
 for the podman process.  This can be disabled by setting the `--http-proxy`
 option to `false`.  The environment variables passed in include `http_proxy`,
 `https_proxy`, `ftp_proxy`, `no_proxy`, and also the upper case versions of
-those.
+those.  This option is only needed when the host system must use a proxy but
+the container should not use any proxy.  Proxy environment variables specified
+for the container in any other way will override the values that would have
+been passed thru from the host.  (Other ways to specify the proxy for the
+container include passing the vaules with the `--env` flag, or hardcoding the
+proxy environment at container build time.)
+
+For example, to disable passing these environment variables from host to
+container:
+
+`--http-proxy=false`
```

Comment:
I was actually thinking about adding an example to the Examples section of each man page, but I think this probably works better.